src/components/film-movie-form/movie-form/index.js

```diff
@@ -1,30 +1,31 @@
 import { Button, FilmFormInputList } from '../../index';
-import { buttonContent } from '../../../constants/const';
-import './movie-form.css';
+import { buttonContent, defaultFilm } from '../../../constants/const';
+import './style.css';
+import { useContext, useState } from 'react';
+import { AppContext } from '../../context';
 
-const MovieForm = ({
-  setFormOpen,
-  setIsMovieAdded,
-  setFilm,
-  film,
-  onSubmit,
-  title
-}) => {
+const MovieForm = () => {
+  const { state, dispatch } = useContext(AppContext);
+  const [film, setFilm] = useState(state.film);
   const handleClick = () => {
```

Comment:
The bracket can be removed.
```suggestion
  const handleClick = () => dispatch({
```

Reply:
done